utils.rb

```diff
@@ -0,0 +1,19 @@
+require 'pathname'
+
+def get_notes_path_from_cli_args()
+  if ARGV.length == 1
+    notes_path = Pathname.new(ARGV[0])
+  else
+    notes_path = Pathname.new("#{ENV['HOME']}/Notes/")
+  end
+  
+  if not notes_path.directory?
+    Kernel.abort("Directory [#{notes_path}] does not exist.
+Please specify your notes directory as a command line argument.
+   ex: ruby todo.md ~/john/my_notes")
+  else
+    puts "notes path: #{notes_path}".inspect
```

Comment:
When run as a cron, this `puts` generates system mail. Maybe remove or come up with a different way to log it?

Reply:
Forgot about this. sorry.

So I guess the philosophy is: we don't do output in the regular flow of the script?
Otherwise we can easily ignore the output in cron jobs like this
`0 * * * * ruby ~/software/obsidian_utils/todo.rb ~/Nextcloud/main > /dev/null 2>&1`

Also, we should add documentation in the README to show how to automate with a cronjob.
Might do it in another PR.